src/fidesops/service/connectors/post_processor_strategy/post_processor_strategy_filter.py

```diff
@@ -0,0 +1,78 @@
+import logging
+from typing import Any, Optional, Dict
+
+from fidesops.schemas.saas.strategy_configuration import (
+    FilterPostProcessorConfiguration,
+    StrategyConfiguration,
+    IdentityParamRef,
+)
+from fidesops.service.connectors.post_processor_strategy.post_processor_strategy import (
+    PostProcessorStrategy,
+)
+
+STRATEGY_NAME = "filter"
+
+logger = logging.getLogger(__name__)
+
+
+class FilterPostProcessorStrategy(PostProcessorStrategy):
+    """
+    Filters object or array given field name and value
+    Value can be reference a dynamic identity passed in through the request OR hard-coded value.
+    E.g.
+    data = [
+        {
+            id: 1397429347
+            email_contact: somebody@email.com
+            name: Somebody Awesome
+        },
+        {
+            id: 238475234
+            email_contact: somebody-else@email.com
+            name: Somebody Cool
+        }
+    ]
+    field: email_contact
+    value: {identity: email}, where email == somebody@email.com
+    result = {
+        id: 1397429347
+        email_contact: somebody@email.com
+        name: Somebody Awesome
+    }
+    """
+
+    def __init__(self, configuration: FilterPostProcessorConfiguration):
+        self.field = configuration.field
+        self.value = configuration.value
+
+    def get_strategy_name(self) -> str:
+        return STRATEGY_NAME
+
+    def process(self, data: Any, identity_data: Dict[str, Any] = None) -> Optional[Any]:
+        """
+        :param data: A list or an object
+        :param identity_data: Dict of cached identity data
+        :return: filtered object or None
+        """
```

Comment:
This is so cool, @pattisdr ! Thanks so much for the detailed explanation, too. I think we can reuse some of your work here, I'll add this into my list of follow-ups.

Reply:
added this detail to the new ticket: https://github.com/ethyca/fidesops/issues/264

Reply:
thank you for reticketing! your scenario is a little tricky because you're sometimes filtering arrays, while `build_refined_target_paths` assumed you'd have the original row (Dict) to start with. to be continued!